webpack.config.js

```diff
@@ -104,38 +104,37 @@ const options = {
                 ],
             },
             {
-                // sync font files referenced by the css to the fonts directory
-                // the publicPath matches the path from the compiled css to the font file
-                // only looks in the fonts folder so pngs in the images folder won't get put in the fonts folder
+                // Copies font files referenced by CSS/JS to the fonts
```

Comment:
@ababic these changes are to bring the project in line with a fix that is on its way into kit, because file-loader is now deprecated. I noticed in this PR because I originally used a background image for the circle, and it wasn't copied correctly to static_compiled. I later found it worked better with a CSS solution.

Reply:
I've realised this part of the MR ought to point at `main` - I'll leave it here but do a separate pr to `main`